docs/superpowers/plans/2026-03-15-renderer-performance.md

```
@@ -0,0 +1,1570 @@
+**Tech Stack:** TypeScript, Electron, Bun test runner (`node:test` + `node:assert/strict`)
+
+**Spec:** `docs/architecture/2026-03-15-renderer-performance-design.md`
+
+**Test command:** `bun test <file>`
+
+---
+
+## File Structure
+
+### New Files
+| File | Responsibility |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)